index.js

```diff
@@ -1,17 +1,51 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {4
```

Comment:
careful, the `4` you left there could make all the tests fail. In this case it does not, but remeber to check for typos